### DISCUSSION TITLE
how set fold

### DISCUSSION THREAD

**Discussion by hnlinzhi19** [nvim-neorg/neorg#841] [General]
When I open a norg file, press 'za' can fold  @document. but when press  'za' heading show not found fold. 
Then i open :h Neorg  document, then move to heading  press 'za' it working .
After this, i copy help documents to my files, but 'za' is not working

**Comment by sveeig** [comment]
I have the same issue. I also use Astronvim, which shows fold markers the signs column - and I can see that the folds are initially there, but once the neorg concealer kicks in and overrides norg syntax symbols, the folds go away.

toggling concealer manually off and on again bring back folds for me:

:Neorg toggle-concealer
:Neorg toggle-concealer

**Reply by hnlinzhi19** [reply]
Thanks ，is a best method for me

**Comment by benlubas** [comment]
Are you using nvim-ufo? This [issue](https://github.com/kevinhwang91/nvim-ufo/issues/33) helped me
